deployment_scripts/deploy_python_as_dir.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# deploy_python_as_dir.sh: deployment script for setting up a project in a frozen directory
```

Comment:
Is `deploy_python_project_as_dir.sh` a better name?

Reply:
Done